src/OpenTelemetry/Trace/Configuration/OpenTelemetrySDK.cs

```diff
@@ -0,0 +1,76 @@
+﻿// <copyright file="OpenTelemetrySDK.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using OpenTelemetry.Trace.Export;
+
+namespace OpenTelemetry.Trace.Configuration
+{
+    public class OpenTelemetrySDK
```

Comment:
Shouldn't this be named:

```suggestion
    public class OpenTelemetrySdk
```

Following:
> When using acronyms, use Pascal case or camel case for acronyms more than two characters long. For example, use HtmlButton or htmlButton. However, you should capitalize acronyms that consist of only two characters, such as System.IO instead of System.Io.

[Source](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN)

Some examples in the BCL like [`MimeMapping` as opposed to `MIMEMapping`](https://referencesource.microsoft.com/#System.Web/MimeMapping.cs,9e3445eb945b84a3). [`IisTracing`](https://referencesource.microsoft.com/#System.Web/IisTraceListener.cs,ca966e01eeb2197b), [`Guid`](https://referencesource.microsoft.com/#mscorlib/system/guid.cs,b622ef5f6b76c10a)

Reply:
Yes changed!